package.json

```diff
@@ -48,8 +48,9 @@
     "glob": "^7.1.1",
     "json-extra": "^0.5.0",
     "lodash.merge": "^4.6.1",
-    "rcs-core": "^2.6.2",
-    "universalify": "^0.1.2"
+    "rcs-core": "^3.0.0-alpha.0",
+    "universalify": "^0.1.2",
+    "yarn": "^1.17.3"
```

Comment:
Please do not add `yarn` as dependency :)

Reply:
Either install `yarn` with `homebrew` (if you are on a mac) or install it globally via npm. I might switch fully to plain npm anyways soon.

Reply:
Sorry, I used to use `npm` and when I've tried install yarn, it shouted that we shouldn't mix package manager, so I've removed the lock file and forgot to check the `package.json` file

Reply:
No worries, all good.